ipaserver/dns_data_management.py

```diff
@@ -37,6 +37,23 @@
     (DNSName(u'_kpasswd._udp'), 464),
 )
 
+IPA_DEFAULT_KDC_URI_REC = (
+    # URI record name, target
+    (DNSName(u'_kpasswd'), u'krb5srv:M:tcp:{server}'),
+    (DNSName(u'_kpasswd'), u'krb5srv:M:udp:{server}'),
+    (DNSName(u'_kerberos'), u'krb5srv:M:tcp:{server}'),
+    (DNSName(u'_kerberos'), u'krb5srv:M:udp:{server}'),
+)
+
+# URI records for KDCProxy must have lower priority than for KDC, clients must
+# prefer to connect directly to KDC
+IPA_KDCPROXY_PRIORITY_PENALIZATION = 10
```

Comment:
I'm shy because with DNS locations we use +50, and users may get lost when I increase it too much